dnsimple/resource_dnsimple_lets_encrypt_certificate.go

```diff
@@ -32,8 +32,9 @@ func resourceDNSimpleLetsEncryptCertificate() *schema.Resource {
 				Optional: true,
 			},
 			"contact_id": {
-				Type:     schema.TypeInt,
-				Required: true,
+				Type:       schema.TypeInt,
+				Optional:   true,
+				Deprecated: "contact_id is deprecated and will be removed in the next major version.",
```

Comment:
To avoid confusion, we should explicitly mention that it has no effect. Or users may be trying to find a replacement and get stuck.

```suggestion
				Deprecated: "contact_id is deprecated and has no effect. The attribute will be removed in the next major version.",
```